db/query/findById.js

```diff
@@ -0,0 +1,12 @@
+const User = require('../models/User');
+
+exports.findUser = (id, done) => {
```

Comment:
CREATE FILE WITH USERqUERY

AND ALL QUESRY RELATED TO uSER SCHEME MOVE THERE

Reply:
created